README.md

```diff
@@ -44,7 +44,7 @@ For using Polly with HttpClient factory from ASP.NET Core 2.1, see our [detailed
 ### Release notes
 
 + The [change log](https://github.com/App-vNext/Polly/blob/master/CHANGELOG.md) describes changes by release.
-+ We tag Pull Requests and Issues with [milestones](https://github.com/App-vNext/Polly/milestones) which match to NuGet package release numbers.
++ We tag Pull Requests and Issues with [milestones](https://github.com/App-vNext/Polly/milestones) which match NuGet package release numbers.
```

Comment:
This makes more sense to me as it was before.

Reply:
_"...correspond to specific..."_?

Without the _to_ it could be read to imply that a single tag can be associated with multiple releases, which is not the case either.